core/ustring.cpp

```diff
@@ -355,24 +357,24 @@ String &String::operator+=(const char *p_str) {
 		return *this;
 	}
 
-	int src_len = 0;
+	int rhs_len = 0;
 	const char *ptr = p_str;
 	while (*(ptr++) != 0) {
-		src_len++;
+		rhs_len++;
 	}
```

Comment:
Any reason not to use `strlen`? CRT  implementations usually have optimizations to take memory layout into account (e.g. https://git.musl-libc.org/cgit/musl/tree/src/string/strlen.c).

Reply:
Thanks for the suggestion! I'm replacing it right now. I don't know why it wasn't used earlier too.